migrator/migrations/m_176_to_m_177_network_baselines_cidr/migration_test.go

```diff
@@ -0,0 +1,197 @@
+package m176tom177
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/google/uuid"
+	"github.com/stackrox/rox/generated/storage"
+	frozenSchemav73 "github.com/stackrox/rox/migrator/migrations/frozenschema/v73"
+	"github.com/stackrox/rox/migrator/migrations/m_176_to_m_177_network_baselines_cidr/networkbaselinestore"
+	"github.com/stackrox/rox/migrator/migrations/m_176_to_m_177_network_baselines_cidr/networkentitystore"
+	pghelper "github.com/stackrox/rox/migrator/migrations/postgreshelper"
+	"github.com/stackrox/rox/migrator/types"
+	"github.com/stackrox/rox/pkg/postgres/pgutils"
+	"github.com/stackrox/rox/pkg/sac"
+	"github.com/stretchr/testify/suite"
+)
+
+func deploymentPeer(id string) *storage.NetworkBaselinePeer {
+	return &storage.NetworkBaselinePeer{
+		Entity: &storage.NetworkEntity{
+			Info: &storage.NetworkEntityInfo{
+				Type: storage.NetworkEntityInfo_DEPLOYMENT,
+				Id:   id,
+				Desc: &storage.NetworkEntityInfo_Deployment_{Deployment: &storage.NetworkEntityInfo_Deployment{}},
+			},
+		},
+		Properties: []*storage.NetworkBaselineConnectionProperties{},
+	}
+}
+
+func externalPeer(id string) *storage.NetworkBaselinePeer {
+	return &storage.NetworkBaselinePeer{
+		Entity: &storage.NetworkEntity{
+			Info: &storage.NetworkEntityInfo{
+				Type: storage.NetworkEntityInfo_EXTERNAL_SOURCE,
+				Id:   id,
+				Desc: &storage.NetworkEntityInfo_ExternalSource_{
+					ExternalSource: &storage.NetworkEntityInfo_ExternalSource{
+						Name:   "provider/region",
+						Source: &storage.NetworkEntityInfo_ExternalSource_Cidr{Cidr: ""},
+					},
+				},
+			},
+		},
+		Properties: []*storage.NetworkBaselineConnectionProperties{},
+	}
+}
+
+var (
+	ctx = sac.WithAllAccess(context.Background())
+
+	ext1Cidr = "42.42.0.0/16"
+	ext2Cidr = "24.24.0.0/16"
+
+	dep1 = uuid.New()
+	dep2 = uuid.New()
+	dep3 = uuid.New()
+	dep4 = uuid.New()
+
+	networkEntities = []*storage.NetworkEntity{
+		{
+			Info: &storage.NetworkEntityInfo{
+				Type: storage.NetworkEntityInfo_EXTERNAL_SOURCE,
+				Id:   "ext1",
+				Desc: &storage.NetworkEntityInfo_ExternalSource_{
+					ExternalSource: &storage.NetworkEntityInfo_ExternalSource{
+						Name: "provider/region",
+						Source: &storage.NetworkEntityInfo_ExternalSource_Cidr{
+							Cidr: ext1Cidr,
+						},
+					},
+				},
+			},
+		},
+		{
+			Info: &storage.NetworkEntityInfo{
+				Type: storage.NetworkEntityInfo_EXTERNAL_SOURCE,
+				Id:   "ext2",
+				Desc: &storage.NetworkEntityInfo_ExternalSource_{
+					ExternalSource: &storage.NetworkEntityInfo_ExternalSource{
+						Name: "provider/region",
+						Source: &storage.NetworkEntityInfo_ExternalSource_Cidr{
+							Cidr: ext2Cidr,
+						},
+					},
+				},
+			},
+		},
+	}
+
+	networkBaselines = []*storage.NetworkBaseline{
+		{
+			DeploymentId: dep1.String(),
+			ClusterId:    "c1",
```

Comment:
I'm surprised you didn't have to make this one a uuid like deployment id.  FWIW, there are constants in `pkg/fixtures/fixtureconsts/fixture_consts.go` for such things.

Reply:
Didn't know about `fixtureconsts`. Updated all entries to use the constants.

Reply:
interesting.  I guess that makes sense as in theory we could change a constant that causes an outdated migration to fail.  So disregard using that for migrations (but at least you know about it for when changing non-migration code)